Source/AssetRipper.SourceGenerated.Extensions/GameObjectExtensions.cs

```diff
@@ -237,19 +239,23 @@ public static IEnumerable<IEditorExtension> FetchHierarchy(this IGameObject root
 
 		public static IReadOnlyDictionary<uint, string> BuildTOS(this IGameObject gameObject)
 		{
-			Dictionary<uint, string> tos = new() { { 0, string.Empty } };
+			Dictionary<uint, string> tos = new();
```

Comment:
What was the reasoning for this change?

Reply:
Side note: I'd be ok with you moving all the code like this into a `TosBuilder` class.

Reply:
Ill just go ahead and revert it since even I'm unsure if the change is necessary lmao.

The reason I believed it is unrequired is because its essentially saying a lookup to a Crc32 of zero.

Given that this is actually possible, id rather revert than it end up causing issues...

Reply:
Given that I couldn't find anything in my Google searches just now, I think it's an acronym from mafaca. I assume it means "table of strings."

Reply:
Taking that into account, I don't like TOS much either. A couple alternative names come to mind:
* StringTable
* StringCache
* StringDictionary

Reply:
It is essentially the same thing as the Crc32 caching taking place in other areas, just a different approach for gameobjects, etc.